docs/docs_src/guide/Model.md

```diff
@@ -107,6 +107,18 @@ User.get({"id": 1}, (error, myUser) => {
 });
 ```
 
+If you are using [`map`](Schema#map-string--string) or [`alias`](Schema#alias-string--string) attribute settings on the key, then you can also use those properties in your call
```

Comment:
I kinda question if documentation changes are necessary here. To me, this is implied behavior. On some level, this is a bug report, and not a feature request. So I'm kinda leaning towards removing this documentation.